.github/workflows/release.yaml

```diff
@@ -54,3 +56,61 @@ jobs:
           --sha256=${{ steps.shasum.outputs.sha }} \
           --url="https://github.com/${{github.repository_owner}}/mob/releases/download/${{ github.event.release.tag_name }}/mob_${{ github.event.release.tag_name }}_darwin_amd64.tar.gz" \
           ${{github.repository_owner}}/homebrew-brew/mob
+
+  publish-arch-linux-mobsh-bin:
+    needs: [release]
+    name: Publish Arch Linux package
+    runs-on: ubuntu-latest
+    env:
+      PACKAGE_NAME: mobsh-bin
+    steps:
+      - name: Create PKGBUILD
+        run: |
+          # Create the output directory
+          mkdir -p ./aur/${{ env.PACKAGE_NAME }}/
+
+          # Strip the leading "v" from the version
+          PACKAGE_VERSION="$(echo "${{ github.event.release.tag_name }}" | sed -e 's/^v//')"
+
+          # Output PKGBUILD
+          cat > ./aur/${{ env.PACKAGE_NAME }}/PKGBUILD << EOF
+          # Where to file issues: https://github.com/${{ github.repository }}/issues
+
+          pkgname=${{ env.PACKAGE_NAME }}
+          pkgver=$PACKAGE_VERSION
+          pkgrel=1
+          pkgdesc="Fast git handover with mob"
+          arch=('x86_64')
+          url="https://github.com/${{ github.repository }}"
+          license=('MIT')
+          depends=("git")
+          optdepends=('espeak-ng-espeak: Multi-lingual software speech synthesizer'
+                      'mbrola-voices-us1: An American English female voice for the MBROLA synthesizer')
+          provides=('mobsh')
+          conflicts=('mobsh' 'mob')
+
+          source_x86_64=("\$url/releases/download/${{ github.event.release.tag_name }}/mob_${{ github.event.release.tag_name }}_linux_amd64.tar.gz")
+          sha256sums_x86_64=("${{ needs.release.outputs.sha }}")
+
+          package() {
+              install -D -m644 "LICENSE" "\$pkgdir/usr/share/licenses/\$pkgname/LICENSE"
+              install -D -m755 "mob" "\$pkgdir/usr/bin/mob"
+          }
+
+          EOF
+
+      # this action is just meant for feedback (how does the file look like?)
+      - name: Publish PKGBUILD for ${{ env.PACKAGE_NAME }}
```

Comment:
You could also store this as an artifact: https://docs.github.com/en/actions/advanced-guides/storing-workflow-data-as-artifacts

Reply:
Thanks! That's a great suggestion.